Userland/Utilities/ping.cpp

```diff
@@ -112,6 +119,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     };
 
     TRY(Core::System::setsockopt(fd, SOL_SOCKET, SO_RCVTIMEO, &timeout, sizeof(timeout)));
+    if (ttl.has_value()) {
+        ErrorOr<void> const set_ttl_result = TRY(Core::System::setsockopt(fd, IPPROTO_IP, IP_TTL, &ttl.value(), sizeof(ttl.value())));
```

Comment:
You're already `TRY`ing `setsockopt`, so you can only get `void` out of this. This means that the `.is_error()` underneath can never be true.

Why not leave it at `TRY` and let the application fail if we could not set the TTL? I'd think that's sensible behavior if something the user requested could not be done.